### PR TITLE
rustfmt-nightly pass after new release of the formatter

### PR DESCRIPTION
**Pull request by Rhuagh**

### CODE REVIEW

File: src/aabb/mod.rs [hunk 1]
```diff
@@ -23,28 +23,24 @@
...
 
 mod aabb2;
 mod aabb3;
 
 pub(crate) fn min<S: PartialOrd + Copy>(lhs: S, rhs: S) -> S {
     match lhs.partial_cmp(&rhs) {
-        Some(Ordering::Less) |
-        Some(Ordering::Equal) |
-        None => lhs,
+        Some(Ordering::Less) | Some(Ordering::Equal) | None => lhs,
```
Comment (kvark) [inline src/aabb/mod.rs]: this doesn't look better to me

File: src/dbvt/mod.rs [hunk 2]
```diff
@@ -291,11 +291,7 @@ enum Node<B> {
 impl<T> DynamicBoundingVolumeTree<T>
 where
     T: TreeValue,
-    T::Bound: Clone
-        + Debug
-        + Contains<T::Bound>
-        + Union<T::Bound, Output = T::Bound>
-        + SurfaceArea,
+    T::Bound: Clone + Debug + Contains<T::Bound> + Union<T::Bound, Output = T::Bound> + SurfaceArea,
```
Comment (kvark) [inline src/dbvt/mod.rs]: this is also a regression

File: src/dbvt/mod.rs [hunk 3]
```diff
@@ -754,17 +746,14 @@ where
...
                 // to the sibling, based on where the old branch node was
                 match self.nodes[parent_parent_index] {
-                    Node::Branch(ref mut b) => {
-                        if b.left == parent_index {
-                            b.left = sibling_index;
-                        } else {
-                            b.right = sibling_index;
-                        }
-                    }
+                    Node::Branch(ref mut b) => if b.left == parent_index {
```
Comment (kvark) [inline src/dbvt/mod.rs]: this also doesn't look better

File: src/dbvt/mod.rs (hunk 2 above)
Comment (Rhuagh) [reply src/dbvt/mod.rs]: Same here as above, no config switch for it, and for this one I agree, looks better with the old version.

File: src/dbvt/mod.rs (hunk 3 above)
Comment (Rhuagh) [reply src/dbvt/mod.rs]: Agree, and there's a config switch for it.

File: src/aabb/mod.rs (hunk 1 above)
Comment (Rhuagh) [reply src/aabb/mod.rs]: There's no config switch for this in rustfmt unfortunately. And I personally think the new version is better, easier to find the right hand of the match arm imho.

**Comment by Rhuagh** [comment]
Updated config and reran.

**Pull request merged**